### PR TITLE
Upgrade to Bootstrap 3

### PR DESCRIPTION
**Pull request by paglias**
**This is a work in progress!**
- [ ] use angular ui directives where possible, not bootstrap plugins (login modal: tabs, accordion for forgotten password, tabs in options)
- [ ] challenges use angular ui modal when updated to bootstrap 3.1.0

### CODE REVIEW

**Comment by colegleason** [comment]
FYI: https://github.com/twbs/bootstrap/releases/tag/v3.1.0

**Comment by paglias** [comment]
Better now than then... I suppose, thanks!

**Comment by paglias** [comment]
@colegleason @lefnire do you think it's better to fix all the conflicts with the develop branch when I'll finish this or it may be a good idea to pull updates from develop into paglias/bs3 occasionally?

**Comment by colegleason** [comment]
I like to rebase my pull request often to try to keep up to date since it
would be silly to work on a modal that got removed or whatever.

On Fri, Jan 31, 2014 at 12:22 PM, Matteo Pagliazzi <notifications@github.com

> wrote:
> 
> @colegleason https://github.com/colegleason @lefnirehttps://github.com/lefniredo you think it's better to fix all the conflicts with the develop branch
> when I'll finish this or it may be a good idea to pull updates from develop
> into paglias/bs3 occasionally?
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/HabitRPG/habitrpg/pull/2636#issuecomment-33828026
> .

## 

## Cole Gleason

Student, University of Illinois at Urbana-Champaign
Email: cg@colegleason.com
Website: colegleason.com

**Comment by lefnire** [comment]
Yeah, me too. `git rebase origin/develop bs3` (fix fix fix) `git push origin bs3 --force`

**Comment by lefnire** [comment]
(unless someone is working on that branch with you)

**Comment by paglias** [comment]
Mmmm... I did `git merge develop` not rebase....

**Comment by lefnire** [comment]
eh, whatever works. rebase makes for a cleaner tree. Merge is the reason some people's PRs come with a bunch of other peoples' commits. It _can_ be dangerous, if a old commit which comes with PR clobbers a newer relevant commit, but usually it's benign - just makes dirtier trees, which I don't personally care much about

**Comment by paglias** [comment]
:+1: 

Will learn rebase the next time (already some conflicts since 2 hours ago...)

I started porting the real app, and since there's a lot of custom css I'll try to review it as I move on, actually I did filters

**Comment by paglias** [comment]
I'm having problems rebasing now after I already merged before

**Comment by colegleason** [comment]
Yeah, that would cause issues since you are including that same content
twice I believe. You could try git rebase -i to remove the merge commits or
you could undo the merge.

On Fri, Jan 31, 2014 at 2:29 PM, Matteo Pagliazzi
notifications@github.comwrote:

> I'm having problems rebasing now after I already merged before
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/HabitRPG/habitrpg/pull/2636#issuecomment-33839362
> .

## 

## Cole Gleason

Student, University of Illinois at Urbana-Champaign
Email: cg@colegleason.com
Website: colegleason.com

**Comment by lefnire** [comment]
once you choose one or the other (merge / rebase), you're kinda locked in and gotta stick to that strategy. Some ways around it (usually what I do):
- `git checkout bs3`
- `git diff develop > bs3.diff`
- `git checkout develop && git branch -D bs3 && git checkout -b bs3`
- `git apply bs3.diff && rm bs3.diff`

Then use rebase & force from then on out

**Comment by paglias** [comment]
Ok, I'm doing a mess, will stay with merge for this one, I'm sorry but I'm having a lot of problems even with merge alone

File: views/shared/modals/drops.jade [hunk 1]
```diff
@@ -12,7 +13,8 @@ div(modal='modals.dropsEnabled')
   .modal-footer
     button.btn.btn-default.cancel(ng-click='modals.dropsEnabled = false')=env.t('close')
 
-div(modal='modals.drop')
+//div(modal='modals.drop')
```
Comment (paglias) [inline views/shared/modals/drops.jade]: @lefnire @SabreCat this modal is not used anymore, right?

**Comment by paglias** [comment]
I believe I'll have it working tomorrow, not finished but "usable", there are still many ugly things, styles to be refactored and a review of how angular ui is used but it works.

If it is possible, (when I'll have it fully working) @lefnire could you deploy this branch so that people can test everything works (I haven't been able to replicate [this](https://github.com/HabitRPG/habitrpg/blob/develop/public/js/controllers/userCtrl.js#L20-L24) for example) because since a lot of JS things changed too I would like to be sure I didn't break anything.

File: views/shared/modals/drops.jade (hunk 1 above)
Comment (SabreCat) [inline views/shared/modals/drops.jade]: Right, at the moment it's been replaced by the green Growl
notifications.http://sabrecat.wordpress.com/

**Comment by paglias** [comment]
Oook, it should be fully working now!

@lefnire I'm not sure this modal is going to work https://github.com/HabitRPG/habitrpg/blob/develop/views/shared/modals/quests.jade#L50, I don't undertand which scope it refers too and in case it's $rootScope where  those variables comes from (party, questHold...)

**Comment by lefnire** [comment]
@paglias `$rootScope.party`. After looking around, `questHold` isn't being handled properly (at all?) so that's a bug, you can ignore it and go on without it

**Comment by paglias** [comment]
@lefnire I will remove it, with questHold as a condition this should have never worked...

**Comment by paglias** [comment]
Ops, it's !questHold...

**Comment by paglias** [comment]
@lefnire with https://github.com/HabitRPG/habitrpg/commit/c663221229c8fb55a4a2632605613a11aa7b91b1 I moved the tasks lists grid from the one using custom code to Bootstrap own grid system.

When using a desktop the differences aren't muc:h

habitrpg your life the role playing game - google chrome_025

vs the old

habitrpg your life the role playing game - google chrome_028

And it should be the same for phones.

While for middle-size devices (I don't know how it looks on Ipad or Android tablets but I think this could be the case) it went from:

habitrpg your life the role playing game - google chrome_030

to 

habitrpg your life the role playing game - google chrome_029

Columns are stacked and not placed 2 on each line with scrollbars.

I think I can achieve the same appareance with some custom code or switch back to the old code but I would like to know if the 2 columns design is used very much or if we can do without it.

**Comment by lefnire** [comment]
So the 2-by is what tablet users see. Last time it was discussed, they really liked it that way - gives them more to work with. But looking at our [demographics](http://gyazo.com/154f24baea983998769dc8a22e1213ca), it seems tablet use is pretty darn rare (I'm looking at 5 & 7). So I'd say keep 2-by if it's easy enough, but if it's a pain ditch it. 

Let's get some other opinions though: @wc8 @SabreCat @lemoness @deilann ? Also, @zakkain was the original author, if he's still around I'd love his opinion!

**Comment by lefnire** [comment]
Also, we can come back to it later if it's difficult now. We can just ref
the commit

**Comment by paglias** [comment]
I managed to get it back, it was easier then I thought with the new grid system allowing for different columns classes for different screen sizes, so no problem.

Anyway @zakkain really made an impressive work, wow!

I finished the whole tasks pages, tomorrow I'll move on the the options

**Comment by lefnire** [comment]
@paglias you are a god.

**Comment by SabreCat** [comment]
I vote :+1: for the single-column layout when cramped. I occasionally need to squash the browser over to a half-screen for something I'm working on, and that 2x2 thing is just an unholy mess.

**Comment by paglias** [comment]
@lefnire it''s done :)

As you can see from the checklist at the top there are still a few minor bugs but nothing really important. 

If you could merge this and deploy to beta so we can ask for real feedback, maybe with a bailey announcement and a link to this issue or a new one to report bugs.

**Comment by paglias** [comment]
As of yesterday it was up to date with develop, unless something changed
during the night that causes a conflict you can safely merged

**Comment by paglias** [comment]
@lefnire about that bug, I have no idea actually, I gave a quick look at it but could find nothing. Will investigate more later

**Comment by lefnire** [comment]
We're on beta, prepare yourself for reports @paglias :) When it comes time to merge to develop, how would you feel if I squashed all the commits into a single commit, applied it all at once? It might make tracking issues to a commit more organized, what with the cross-merging on this branch. Any concern it would lose commits you wanted to track individually?

**Comment by deilann** [comment]
Interested in this convo myself, for ticket shuffling.

note to self: remark on "report an issue" ticket to state whether you are
on beta or prod.

**Comment by deilann** [comment]
Yes, but I've seen people post issues in the past that I knew were a beta
issue, but they didn't mention it in the issue ticket.

**Comment by wc8** [comment]
So, are we reporting "funkiness" in this thread directly or just labeling _beta_ and @paglias ?

minor:
- [ ] dailies text is right aligned, so in _narrow windows_, the info icons (streaks, checklist counter, delete, edit, tags etc.) pushing the daily text out of view. 
  Update, it's not all the text out of view, it's the streak in one case.
- [x] avatars low on mounts, mounts low and squished to pets,

(Firefox, Mac, beta.habit)

**Comment by wc8** [comment]
Can't all be bug reports, @paglias :

• I like the blue line chat highlight in beta.
• Ooh, and the resizable chat box!
• hooray, I think this fixed the Rest in Tavern button visibility issue #1350 !

**Comment by deilann** [comment]
Like the new tag setup (horizontal scrolling = :() but if you have a LOT of
tags, they take up multiple lines, also not good. Maybe a vertical scroll?

**Comment by deilann** [comment]
**FIXED** I cannot tell if "slim" or "broad" is selected on my profile settings.

Also like the resizable chat box. Do not like the new font for the name
tags. Hard to read, seems off-center.

Quest info on the left no longer condenses with a scroll bar.

**Comment by CortJstr** [comment]
I also have the low avatar/squished pet problem (Firefox 27/Win7) and all of my pets lost their progress towards being mounts. The progress bars are all at 0 and feeding them the preferred food produces the "really likes" message but the bar stays where it is.

**Comment by deilann** [comment]
**FIXED** GM star is black in the Hall, making it tough to see on purple/blue-green.
Names in the Hall of Patrons do not show up unless they are a contributor.

**Comment by deilann** [comment]
@CortJstr Fear not, they haven't lost their progress -- it's just not
showing.

Same problem here, and if a pet exists, it shows up lower than the
pixelpaw, creating really long rows.

**Comment by deilann** [comment]
CDS input box goes across the screen, making the :00 afterwards on the next
line.

And I think that's all I've found!

**Comment by lefnire** [comment]
Oh my god paglias, I just did a code review of the changes... How long did this take you? This is a huge changeset!

**Comment by wc8** [comment]
- [x] white on white for grey-level users when typing @ Tavern chat.

**Comment by deilann** [comment]
@wc8 I imagein that's what I'm seeing in The Hall of Patrons.

**Comment by colegleason** [comment]
Fantastic work! It looks really nice! I went through and took some screenshots of some things that could use fixing. Mostly minor, and some were mentioned above.

CDS (via @deilann)
screen shot 2014-02-07 at 10 08 36 pm
The nametag here seems a bit low (not vertically centered).
screen shot 2014-02-07 at 10 07 03 pm
**FIXED** Link is unreadable.
screen shot 2014-02-07 at 10 06 50 pm
Challenges seem to be larger than needed. Is that bottom white bar intended?
screen shot 2014-02-07 at 10 07 39 pm
Edit Group button is off.
screen shot 2014-02-07 at 10 06 42 pm
**FIXED** Pet feeding bars don't work (mentioned by @CortJstr)
screen shot 2014-02-07 at 10 08 09 pm
The send chat and fetch messages button don't seem to be aligned to the right of the chat box. Also the fetch messages button is smaller than the send chat button.
screen shot 2014-02-07 at 10 06 20 pm
It would be awesome to have a little bit more padding between nametags and the next line. Alternatively, make nametags thinner.
screen shot 2014-02-07 at 10 06 11 pm
Autocomplete name popup obscures text in chat.
screen shot 2014-02-07 at 10 06 04 pm

**Comment by lefnire** [comment]
Fix for challenge bodies [started here](https://gist.github.com/lefnire/ccc31589ae879921cce3), but it causes the browser to freeze.. no clue :o. Anyway, pushing fix for mounts, and pet-feeding

**Comment by paglias** [comment]
@lefnire wow that's a lot, I'll give a look at everything later. Maybe panel-collapse needs a dot in front of it, or it is an element?

**Comment by wc8** [comment]
one more thing on Firefox mobile (which works well, btw) there are some header alignment issues which nigh also affect narrow browsers on desktop,
Such as Bailey peeking between your mana bar and your mana points.

**Comment by paglias** [comment]
@wc8 could you check that header on mobile thing works well on production ?
Because I didn't touch the header I believe
Il 08/feb/2014 06:53 "wc8" notifications@github.com ha scritto:

**Comment by lefnire** [comment]
Alright, we're merged to develop. Master now has a snapshot of what develop _was_, and I'll be doing master=>production, develop=>beta over coming days. Just want to reduce near-future merge conflicts & such.

**Comment by brightbold** [comment]
**FIXED** In Chrome, if people have +1ed a post, the +N doesn't appear unless you hover over the space where it should be and +1 it yourself. Then it shows up. Unplussed posts work properly (when you hover over the space you see the +1.

And FWIW, I don't use tablet widths often but agree with @SabreCat that the 2-up layout is kind of a pain (although for me it's less the two across design than cutting off the columns vertically after only a task or two.) I like the one-column design. Maybe anchor links to take you to the top of each section would satisfy the people who prefer the 2-up?

**Comment by brightbold** [comment]
Hmm. Couldn't buy the gryphon quest on beta. When I click on it, I get the blue outline, but nothing else happens. Works fine not on beta.

**Comment by paglias** [comment]
1. https://github.com/HabitRPG/habitrpg/commit/0fe0cb4f2d97be42dc0dd6c3a1b670f5765973be Fixes various issues regarding labels (+1, badges, dark star in hall...) and buttons.
2. Another issue I found is that the form for awarding contrib levels & co has some strange sizes.
3. @brightbold I tried just now and I can buy quests without problems, you don't even see the modal window with the description?
4. About the tablet layout we can easily switch to 1-column layout if you want (cc @lefnire )
5. @colegleason The different size for the chat box buttons is because one (Send Chat) in an `input` element with some classes applied to it, the sync one is a `button` element. We may change the input to be a button but it wouldn't send chat messages on ENTER.
6. @deilann 'Quest info on the left no longer condenses with a scroll bar.
   ' can you post a screenshot? In general modals no longer have a fixed height, so you'll have to scroll to the bottom for the buttons, I changed this only for bailey's modal because it would have been very annoying given it's length.
7. @wc8 dailies text right aligned, could you post a screenshot?

**Comment by paglias** [comment]
@deilann cds input, I have it half page, still to large but it shouldn't be all the page

tags, would you prefer them on a new line after a certain point?

**Comment by paglias** [comment]
@colegleason the white strip under the challenge: it seems like the collapse class isn't applied...

**Comment by paglias** [comment]
@wc8 

bailey align on mobile is the same as before:

**Comment by lefnire** [comment]
@paglias things are looking fantastic per your recent commits! This ticket is pretty unwieldy, if it helps you sometimes on tickets like this I'll move comments into the checklist at the top and delete the comment.
1. I also can't buy quests (from the market, clicking [gives error](http://gyazo.com/53528148e8a2fe812d6f93638700083a))
2. 1 or 2 -col layout: sounds like people prefer 1-col. @wc8 should we do a poll?
3. " the white strip under the challenge" here's [my attempt](https://gist.github.com/lefnire/ccc31589ae879921cce3), but it freezes the UI. will try again tomorrow or monday, unless you beat me to it.
4. "use angular ui directives where possible": I think we can do this incrementally and not make a requirement for this release

**Comment by SabreCat** [comment]
Minor thing, visible in some of the screenshots above: the "Lvl X" tag on avatars is cut off on the right and bottom sides. You can tell from the remaining corner, which is the only one still appearing as rounded. Might be Firefox-specific.

**Comment by deilann** [comment]
re: quest info:

I can't now, because we're no longer in a quest. I'm not talking about the
modal; I'm talking about when the quest info shows up on the left of the
part page.

As for the CDS, it might not have been the whole page, but it was big
enough to knock the :00 to the next line, which looked very silly.

As for the tags, they DO show up on multiple lines for me -- but I've got
four lines of tags! so if maybe after two lines you get a scrollbar, that
would be nice.

**Comment by paglias** [comment]
@lefnire 
1. buyQuest should be fixed now
2. I found what questHold was used to, it is set to true if you click "Ask Later" on a quest invitation. But given that it was a simple property of InventoryCtrl's scope I don't think it got saved anywhere. Do you know if quest invitation were ever shown?
3. 1-2 columns, you don't like the two columns layout at all or the fact that columns have fixed width and scrolling bars?
4. the white strip should disappear when the collapse class is applied but it is never applied to it and even with it a really small strip remains. I think your attemp may fail because `panel-collapse` is a class not an element (https://gist.github.com/lefnire/ccc31589ae879921cce3#file-challenge-bodies-diff-L11)
5. "use angular ui directives where possible" yes it is not necessary for this release (I think only the paypal buttons prevent us from releasing it) and I don't know how much more can be migrated to angular-ui

@SabreCat it's not a bug :) I thought it was nicer with only the top left border rounded. I can revert it if you want.

@deilann Understood, do you want a fixed height and scrollbars?

**Comment by deilann** [comment]
Yes please! (It gets ten times worse if you zoom, which I often need to, so
a fixed height would be greatly appreciated!)

**Comment by paglias** [comment]
Added to checklist

**Comment by SabreCat** [comment]
@paglias I like it all rounded. If you dislike the rounded corners, we could try removing the border radius entirely, see how it feels squared off. But one rounded corner and three square ones just makes it look like it's off-center.

Additional tiny thing: The Social subtabs with icons need spaces between the icon and the caption.

**Comment by paglias** [comment]
@SabreCat ok, I thought of it as popping out from the bottom-right corner, as a card hidden under something with only one corner shown.

I'll fix both in a moment.

**Comment by SabreCat** [comment]
Text in the badge bubbles for task checklists doesn't look centered correctly:

screenclip

There's extra space on the right-hand side, and at least to my eye it looks like an inactive Daily's checklist bubble text isn't vertically centered while the active Daily's is.

It'd also be cool if the bubble itself were vertically centered in line with the other icons (note how it's flush with the top of the "tags" but extends beyond their base), but I think that's the way it's always been.

**Comment by SabreCat** [comment]
@paglias: The "tucked card" look works when there's a border around the herobox, as is the case with the logged-in user (and even there, it looks like there's about 1px gap between the lvl tag and the right border). But party members don't have borders, so the "card" is tucked behind nothing.

**Comment by paglias** [comment]
Fixed some time ago, needs to go live, for the vertical centering I think
they are in the same position: both a bit too low

2014-02-08 19:55 GMT+01:00 Sabe Jones notifications@github.com:

> Text in the badge bubbles for task checklists doesn't look centered
> correctly:
> 
> [image: screenclip]https://f.cloud.github.com/assets/4501321/2118071/594fbdd6-90f2-11e3-877c-0aa24cd02937.png
> 
> There's extra space on the right-hand side, and at least to my eye it
> looks like an inactive Daily's checklist bubble isn't vertically centered
> while the active Daily's is.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/HabitRPG/habitrpg/pull/2636#issuecomment-34552329
> .

## 

Matteo Pagliazzi - paglias.net

**Comment by SabreCat** [comment]
Tags icon of a task in edit mode crashes into the "cancel" X:

screenclip

**Comment by wc8** [comment]
dailies alignment
To replicate you have to have a checklist, a streak and get the browser narrow, but not narrow enough to not trigger the switch from 4 columns to 2x2.

@lefnire :+1: 2x2 poll. It's a way of announcing it beforehand and double-checking that there aren't a lot of users out there who love that view but haven't seen this thread.

**Comment by paglias** [comment]
@wc8 I think this is a very edge case....

2014-02-08 23:18 GMT+01:00 wc8 notifications@github.com:

> [image: dailies alignment]https://f.cloud.github.com/assets/3326182/2118530/39ee3f5a-910e-11e3-813d-be3ca850c50e.png
> To replicate you have to have a checklist, a streak and get the browser
> narrow, but not narrow enough to not trigger the switch from 4 columns to
> 2x2.
> 
> @lefnire https://github.com/lefnire [image: :+1:] 2x2 poll. It's a way
> of announcing it beforehand and double-checking that there aren't a lot of
> users out there who love that view but haven't seen this thread.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/HabitRPG/habitrpg/pull/2636#issuecomment-34558212
> .

## 

Matteo Pagliazzi - paglias.net

**Comment by wc8** [comment]
@paglias hence "minor" listing. I just happened to have my browser sitting at that width when I tested, since then I have simply sized it wider.

**Comment by paglias** [comment]
The fact is, what should be the behavior in this case? With old
bootstrap (production) it is anything different?

2014-02-09 0:36 GMT+01:00, wc8 notifications@github.com:

> @paglias hence "minor" listing. I just happened to have my browser sitting
> at that width when I tested, since then I have simply sized it wider.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/HabitRPG/habitrpg/pull/2636#issuecomment-34560147

## 

Matteo Pagliazzi - paglias.net

**Comment by wc8** [comment]
Sorry for the confusion, swamped this weekend. I meant to add, this was one thing I forgot to check against beta, could very well exist in main section. I can't take the time to follow up at the moment, sorry. 
Otherwise, overall, just wanted to note that the site seems more responsive & snappy over all.

**Comment by SabreCat** [comment]
The "strong" emphasis used for "Unallocated Attribute Points" in Character Build spills over into the next table cell and the help popovers:

screenclip

**Comment by brightbold** [comment]
Not sure if this is related to beta (let me know if not and I'll file a separate issue) but my quest damage ran twice this morning:
image

I do keep two Habit tabs open (one for chat and one for tasks) so that's possibly a culprit? But I've been doing that for a while and this has never happened before.

**Comment by paglias** [comment]
@SabreCat fixed
@brightbold I don't think it can related to this

**Comment by paglias** [comment]
@deilann fixed height challenges panel -> done

**Comment by brightbold** [comment]
Was it an intentional design decision that the column headers (Habits, Dailies. etc.) are no longer bold? I liked them better when they stood out from the other text, so if it was intentional, I'd recommend we switch the font-weight to 300 so they're lighter than the other text. Or 700 is good too if this was an accidental change.

**Comment by paglias** [comment]
@brightbold I personally like they're not bolder.

Anyway it was not intentionaò, it's part of the different feel that Bootstrap 3 ha introduced, something lighter than the previous version.

**Comment by lefnire** [comment]
@paglias things are really coming together. Are there any remaining critical bits before I bring this to master? I'm gonna work on the PayPal thing today

**Comment by paglias** [comment]
@lefnire no critical bugs. I would like to make some forms nicer but that's not important

**Comment by brightbold** [comment]
@paglias I like the less heavy look as well. But I'd go lighter on the headers in that case - `font-weight: 100` or 300. Just to add some visual differentiation and make it a little more stylish. Obviously low priority though.

**Comment by paglias** [comment]
Oh lighter, I thought bolder, sorry.

I tried with 300 and they look fantastic (we're at 500 now and at 100 they appears the same). I'll commit that immediately!

**Comment by brightbold** [comment]
Yeah I think we may not have the 100 weight loading so it just uses 300 instead. Glad you like that better!

**Comment by lefnire** [comment]
> see if we can remove jquery and use only angular.element

We could, but not for sorting. I've also done a lot of work with sortable directives in angular (I'll eventually replace our sortable with [this](https://github.com/angular-ui/ui-sortable), but super low prio future item) and I haven't found any solutions out there that aren't jquery-ui based, so I think we're stuck with jquery for now.

> paypal buttons breaks modal template

Fixed! Cool beans everyone, unless there are objections I'll be merging soon

**Comment by paglias** [comment]
And angular-ui sortable requires jquery so let's stick with it.

Anyway I don't think that using jquery impacts performances or memory usage, if we could do without it, better, but if it requires so much work as it seems I think we have more important things to do.

Let's merge :)

**Comment by lefnire** [comment]
live!

Small bits: funky spacing [in menu](http://gyazo.com/42d9c4e7b49310726d5eb3da7034b198) & [wallet](http://gyazo.com/3832e645a1d9027e77388e962635c963)

**Comment by lemoness** [comment]
This is live on Prod, right? So I can put it in the status report?

**Comment by lefnire** [comment]
Yep!

**Comment by paglias** [comment]
Debug buttons in production... (Note to self)

**Comment by benmanley** [comment]
@paglias I've amended the alignment of the task meta icons. Should fix some of the issues mentioned above by @SabreCat, plus a couple of other discrepancies. #2813

**Comment by paglias** [comment]
@benmanley thanks, merged!

@lefnire `funky spacing` fixed

**Comment by SabreCat** [comment]
The Rebirth prestige tag is looking rather smushed. Needs some padding vs. the text:

screenclip

**Comment by paglias** [comment]
@SabreCat fixed

**Pull request merged**